npm/private/lifecycle/bundle.sh

```diff
@@ -13,8 +13,4 @@ set -o errexit -o nounset
 # This avoids users having to fetch all those packages just to run the postinstall hooks.
 
 npm install
-npx -y @vercel/ncc@0.33.4 build lifecycle-hooks.js -o min
-# ascii_only avoids bad unicode conversions, fixing
-# https://github.com/aspect-build/rules_js/issues/45
-npx -y terser@5.12.1 min/index.js -b ascii_only=true > min/index.min.js
```

Comment:
Modifying the file makes sourcemaps not work. I wasn't able to reproduce the issue that this solves.